testing/tests/DevExpress.viz.charts/chart.part7.tests.js

```diff
@@ -2086,4 +2087,82 @@ $('<div id="chartContainer">').appendTo('#qunit-fixture');
         assert.strictEqual(this.labels[2].draw.callCount, 0);
         assert.strictEqual(this.labels[3].draw.callCount, 0);
     });
+
+    QUnit.module('Render graphic objects', $.extend({}, commons.environment, {
+        beforeEach: function() {
+            commons.environment.beforeEach.call(this);
+            this.clock = sinon.useFakeTimers();
+
+            this.fakeGraphicObjects = sinon.stub(graphicObjects, 'getGraphicObjects', function() {
+                return {
+                    'id_1': { element: 'linear', colors: 'colors_1', rotationAngle: 30 },
+                    'id_2': { element: 'radial', colors: 'colors_2' },
+                    'id_3': { element: 'pattern', template: () => {}, width: 20, height: 10 },
+                    'id_4': { element: 'incorrect_type' }
+                };
+            });
+        },
+        afterEach: function() {
+            this.clock.restore();
+            commons.environment.afterEach.call(this);
+            this.fakeGraphicObjects.restore();
+        },
+    }));
+
+    QUnit.test('Should create graphic objects', function(assert) {
```

Comment:
let's name tests more concrete. When should it create graphic objects?
```suggestion
    QUnit.test('Should create graphic objects on charts creating', function(assert) {
```